src/main/java/com/vso/model/entity/Photo.java

```diff
@@ -9,12 +10,13 @@ public class Photo {
 
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
-    private long id;
+    @Column(name = "id")
+    private Integer id;
     @Column(insertable = false,updatable = false)
-    private long user_id;
-    @Column
+    private Integer user_id;
```

Comment:
Why you have this column?

Reply:
it was there, I only change the data type from long to Integer.